main.js

```diff
@@ -0,0 +1,28 @@
+let gridImages = document.getElementById("grid");
+let gen_btn  = document.getElementById("image-btn");
+let numImagesInput = document.getElementById("num-images");
+
+function fetchImages(numImages) {
+  fetch(`https://jsonplaceholder.typicode.com/photos?_limit=${numImages}`)
+  .then((response) => response.json())
+  .then((result) => {
+    gridImages.innerHTML = "";
+    result.map((value) => {
+      let key = value.url;
+      let imgTag = document.createElement("img");
+      imgTag.src = ` https://picsum.photos/160/?random=${key}`;
+      gridImages.appendChild(imgTag);
+    });
+  });
+}
+
+fetchImages(3);
+
+gen_btn.addEventListener("click", () => {
```

Comment:
Change the logic, so that old images are not removed and new images are added after them